### ISSUE TITLE
A question about label transfer

### ISSUE THREAD

**Issue by AprilYuge** [ChaozhongLiu/MinNet#1]
Hi Chaozhong,

I read the MinNet paper and have a question about the way you performed label transfer from RNA to ATAC using MinNet. It seems to me that MinNet requires labels from both modalities at least when training the network and you also emphasized that MinNet is a supervised network. Then how can you perform label transfer when you don't have any label information for one modality? 

Since you assessed the performance of MinNet on label transfer in your paper. Did you just assess that on the testing data using MinNet trained on labelled data from similar tissues? Does this mean in real practice, when I want to transfer label from an existing annotated scRNA-seq to scATAC-seq data or to annotate a new scATAC-seq data, I have to first find both **labelled** scRNA-seq and **labelled** scATAC-seq data from similar tissues and use them to train MinNet?

Thank you!

**Comment by ChaozhongLiu** [comment]
Hi there,

1. ``Did you just assess that on the testing data using MinNet trained on labelled data from similar tissues? ``
    Yes. Model trained on BMMC, testing sets were BMMC and PBMC.  

2. ``I have to first train MinNet?``
Yes, training is needed when applying to different tissues. The BMMC model is available on GitHub, it can be used in BMMC and PBMC datasets. But for quite different ones, training is needed. 

3. ``find both labelled scRNA-seq and labelled scATAC-seq data from similar tissues``
To be exact, the training dataset is from single-cell multiome sequencing techniques. So if scRNA-seq is labelled, there is no need to label scATAC-seq since cells are the same in the two datasets.

Please let me know which tissue you are working with, I can help find the training dataset and train the model for you if you are not familiar with it.

BTW, MinNet is super useful when there is batch effect in datasets you want to integrate. Otherwise, I'd recommend [GLUE](https://scglue.readthedocs.io/en/latest/).

**Comment by AprilYuge** [comment]
Hi Chaozhong,

Thank you for the reply. I still have a question for the third point. I think MinNet is designed for unpaired single-cell data. Then, why do I need to train it on paired multiomic data?

Best,

Yuge

**Comment by ChaozhongLiu** [comment]
Hi Yuge,

It's trained on multiome data, so that after training, it can be applied to any unpaired data from similar tissues.
I'm not sure if this answers your question. Let me know.

Chaozhong

**Comment by AprilYuge** [comment]
But the training doesn't require/use the pairing information right? Or in other words, can I train MinNet using unpaired scRNA-seq and scATAC-seq from the same tissue?

**Comment by ChaozhongLiu** [comment]
The pairing information is used in computing the contrastive loss. You can check to the Methods section ``The Siamese neural network`` in the paper for details.  

Paired cells are positive pairs in contrastive loss, different cells are negative pairs.

**Comment by AprilYuge** [comment]
I see! Thank you for the clarification.

**Comment by ChaozhongLiu** [comment]
No problem!
